server/examples/go/helloworld/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+	"app_id": "helloworld",
+	"display_name": "Hello, world!",
+	"app_type": "http",
+	"root_url": "http://localhost:8080",
```

Comment:
Homepage URL will be needed to create the OAuth app, so better put it in the exaple manifest.